plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/Symbol.java

```diff
@@ -144,25 +146,26 @@ public void setIconRotate(Float value) {
   /**
    * Get the IconOffset property
    *
-   * @return property wrapper value around Float[]
+   * @return PointF value for Float[]
    */
-  public Float[] getIconOffset() {
+  public PointF getIconOffset() {
     JsonArray jsonArray = jsonObject.getAsJsonArray("icon-offset");
     Float[] value = new Float[jsonArray.size()];
     for (int i = 0; i < jsonArray.size(); i++) {
       value[i] = jsonArray.get(i).getAsFloat();
     }
-    return value;
+    return new PointF(value[0], value[1]);
   }
 
   /**
    * Set the IconOffset property.
    * <p>
    * To update the symbol on the map use {@link SymbolManager#update(Annotation)}.
    * <p>
-   * @param value constant property value for Float[]
+   * @param pointF value for Float[]
    */
-  public void setIconOffset(Float[] value) {
+  public void setIconOffset(PointF pointF) {
+    Float[] value = new Float[]{pointF.x, pointF.y};
```

Comment:
This seems like an unnecessary object allocation, we could insert `pointF.x` and `pointF.y` to the JSON array manually as we know there will always be 2 values, no need to iterate through them.